### PR TITLE
Fix TypeError by converting axis to a tuple (Small fix)

### LINKED ISSUE #14977: [Bug]: TypeError when converting axis to tuple. Not the correct way to initialise a tuple
### Bug Explanation

I encountered a TypeError while attempting to convert the `axis` variable to a tuple. The error occurred because `axis` was initially assigned an integer value, which is not iterable. [Link](https://github.com/unifyai/ivy/blob/master/ivy/functional/backends/jax/linear_algebra.py#L199) 

### Steps to Reproduce Bug

Call the `matrix_norm` function and pass `axis` as any integer.

### Environment

Windows 11, VSCode, conda python interpreter. 

### Ivy Version

Latest master

### Backend

- [ ] TensorFlow
- [ ] NumPy
- [ ] PyTorch
- [X] JAX

### Device

_No response_

### PR DESCRIPTION
**Pull request by iammarco11**
Converted the variable axis from an integer to a tuple to resolve the TypeError. The original code was throwing a 'int' object is not iterable error. By encapsulating axis with parentheses and adding a comma, it was transformed into a tuple, allowing it to be used as an argument in functions that require an iterable object.

Closes #14977

### CODE REVIEW

**Comment by iammarco11** [comment]
@RickSanchezStoic since you were assigned to the issue #14977 would you like to review and share some inputs on this PR?

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @iammarco11 

Thanks for the PR and spotting this isssue,
Would you check the comments below.

**Comment by iammarco11** [comment]
@hirwa-nshuti I have made the changes you requested can you take a look at it again?

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!

Thanks for contributing
